### DISCUSSION TITLE
Forwarding to more than one email address

### DISCUSSION THREAD

**Discussion by WhamboMPS** [simple-login/app#574] [Kind words ❤️]
Love that SimpleLogin allows forwarding of inbound messages to more than one mailbox. This is a key feature for couples who don't manage their own email infrastructure and therefore can't create distribution groups. Thanks!

**Comment by nguyenkims** [comment]
@WhamboMPS  Thanks for the kind words! This multi mailbox support is also quite handy for accounts that are shared between people, for example an ecommerce account which is used by family members.
